code/__DEFINES/antagonists.dm

```diff
@@ -241,6 +241,9 @@ GLOBAL_LIST_INIT(ai_employers, list(
 /// Checks if the given mob is a malf ai.
 #define IS_MALF_AI(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/malf_ai))
 
+/// Checks if the given mob is a slasher.
+#define IS_SLASHER(mob) (mob?.mind?.has_antag_datum(/datum/antagonist/slasher)) //MONKESTATION EDIT
+
```

Comment:
This can actually go in `code\__DEFINES\~monkestation\antagonists.dm`, where you don't need to mark it as a monkestation-specific item.